src/pages/User.vue

```diff
@@ -11,19 +21,35 @@ import { useRoute } from 'vue-router'
 import { getFirstParam } from '/@/lib/params'
 import apis, { ItemSummary } from '/@/lib/apis'
 import useTitle from './use/title'
+import UserIcon from '/@/components/UI/UserIcon.vue'
+import ItemFlexList from '/@/components/Item/ItemFlexList.vue'
+import CommentGrid from '../components/UserPage/CommentGrid.vue'
 
 export default defineComponent({
   name: 'User',
+  components: {
+    UserIcon,
+    ItemFlexList,
+    CommentGrid
+  },
   setup() {
     const route = useRoute()
 
     const state = reactive({
       username: computed(() => getFirstParam(route.params.name) ?? ''),
-      items: [] as ItemSummary[]
+      items: [] as ItemSummary[],
+      comments: [] as { text: string; item: ItemSummary }[]
     })
     watchEffect(async () => {
-      const { data } = await apis.getItems(state.username)
-      state.items = data
+      const [{ data: items }, { data: comments }] = await Promise.all([
+        apis.getItems(state.username),
+        apis.getComments(state.username)
+      ])
+      state.items = items
+      state.comments = comments.map(({ text, item }) => ({
+        text,
+        item
+      }))
```

Comment:
```suggestion
      state.comments = comments
```
ここ(ほぼ)何もしてないのでそのまま代入してよいですね 🤘